sdk/build.gradle

```diff
@@ -89,17 +88,27 @@ android {
     useLibrary 'org.apache.http.legacy'
 }
 
+ext {
+    playServicesVersion = '11.8.0'
+    androidSupportVersion = '27.0.2'
+}
+
+configurations.all {
+    resolutionStrategy.force "com.android.support:support-v4:${androidSupportVersion}"
```

Comment:
Is this the way to solve the compatibility issues which we talk the last day about?

Reply:
Yes, that's telling Gradle to force a version of the dependency.